toqito/helper/np_array_as_expr.py

```diff
@@ -6,8 +6,7 @@
 
 
 def np_array_as_expr(np_arr: [np.ndarray]) -> Expression:
```

Comment:
Hmm. What's our comment docstring convention? Should we begin the comment at the end of the `"""` or start a new line?

I think either is okay, but we should be consistent. The pedantic part of me wants to go through and do the former everywhere, but that's another issue for another time :)

Reply:
How about we leave this for #280? These types of inconsistencies are currently being caught locally.